components/ProfilePhoto.jsx

```diff
@@ -18,9 +18,9 @@ export default function TradingCardImage({ imageUrl, weeklyCapacity, forecastedH
           />
         </div>
       </div>
-      <div className={`cmp-trading-card-image__hours-container cmp-trading-card-image__hours-container--${classColorModifier}`}>
+      <div className={`cmp-profile-photo__hours cmp-profile-photo__hours--${classColorModifier}`}>
         <span aria-hidden="true">
-          <span className="cmp-trading-card-image__hours">{weeklyCapacity - forecastedHours}</span>
+          <span className="cmp-profile-photo__hours-span">{weeklyCapacity - forecastedHours}</span>
```

Comment:
what about something like `cmp-profile-photo__hours-text` instead `cmp-profile-photo__hours-span`? There is more than one `<span>`  inside the `cmp-profile-photo__hours` block so it would make it clear the class refers to one specific `<span>`